src/oc-id/app/views/layouts/application.html.erb

```diff
@@ -14,18 +14,18 @@
       <ul class="title-area">
         <li class="name">
           <h1>
-            <%= link_to '', root_path, :class => 'logo default' %>
+            <%= chef_logo(:a, :href => root_path, :class => 'logo').html_safe %>
           </h1>
         </li>
         <li class="toggle-topbar menu-icon"><a href="#"><span></span></a></li>
       </ul>
-      <div class="top-bar-section">
+      <div class="top-bar-section main">
         <ul class="left">
           <% if signed_in? %>
             <li><a>Signed in as: <%= current_user.username %></a></li>
```

Comment:
This should probably be a:

```erb
<%= link_to root_path do %>
  Signed in as: <%= current_user.username %>
<% end %>
```

because we'll want to to link to the profile once that's in place.